lib/ruby_lsp/server.rb

```diff
@@ -577,12 +582,12 @@ def workspace_did_change_watched_files(message)
 
         case change[:type]
         when Constant::FileChangeType::CREATED
-          @index.index_single(indexable)
+          @global_state.index.index_single(indexable)
```

Comment:
NIT: we can put the index in a local variable and then avoid doing `@global_state.index` in every branch.